### PR TITLE
Update integration test process

### PR DESCRIPTION
**Pull request by rjschwei**
Include tests for LTSS and update processes to check registry access and configuration.

### CODE REVIEW

File: integration_test-process.txt [hunk 1]
```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
...
+  + systemctl start docker.service
+  + systemctl status docker.service
+    - Service should be running
+  + docker pull suse/ltss/sle$VERSION/bci-base
+    - Should pull the container from registry-$FRAMEWORK.susecloud.net
+  + systemctl stop docker.service
+  + zypper in podman
+  + podman pull suse/ltss/sle$VERSION/bci-base
+- zypper migration
+  + Should fail because LTSS repos are active, and possibly othre modules
```
Comment (jesusbv) [inline integration_test-process.txt:177]: s/othre/other

File: integration_test-process.txt [hunk 2]
```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
...
   + Expected to succeed
-- With the new cloud-regionsrv-client code in place run "zypper migration"
+  With the new cloud-regionsrv-client code in place
+- registercloudguest --clean
+- registercloudguest -r LTSS_REG_CODE
+  + Expected to fail because the underlying system base product is not
+    registered
+- registercloudguest
+  + no error
+  + sucees message on stdout
```
Comment (jesusbv) [inline integration_test-process.txt:160]: s/sucees/success

File: integration_test-process.txt [hunk 3]
```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
...
+  + $? is 0
+- registercloudguest -r LTSS_REG_CODE
+  + zypper lr | grep LTSS has LTSS repositories
+  + zypper ref
+  + zypper up
+    + Expected to have new updates due to the addition o fthe LTSS repos
```
Comment (jesusbv) [inline integration_test-process.txt:166]: s/o fthe/of the

Comment (rjschwei) [reply integration_test-process.txt:166]: Fixed, thanks

File: integration_test-process.txt [hunk 4]
```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
...
+  + source /etc/profile.local
+  + systemctl start docker.service
+  + systemctl status docker.service
+    - Service should be running
+  + docker pull suse/ltss/sle$VERSION/bci-base
+    - Should pull the container from registry-$FRAMEWORK.susecloud.net
+  + systemctl stop docker.service
```
Comment (jesusbv) [inline integration_test-process.txt:201]: can we not run podman while docker is running ? I dont remember if it raised a socket error

File: integration_test-process.txt (hunk 2 above)
Comment (rjschwei) [reply integration_test-process.txt:160]: Fixed thanks

File: integration_test-process.txt (hunk 1 above)
Comment (rjschwei) [reply integration_test-process.txt:177]: Fixed thanks

File: integration_test-process.txt [hunk 5]
```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
...
+  + zypper in podman
+  + podman pull suse/ltss/sle$VERSION/bci-base
+- zypper migration
+  + Should fail because LTSS repos are active, and possibly othre modules
+  + Use the command presented to remove the product/modules
+- zypper migration
+  + Migrate to the highest available SP
+  + Expected to succeed
+- zypper migration
+  + Should fail because LTSS repos are active, and possibly othre modules
```
Comment (jesusbv) [inline integration_test-process.txt:211]: ditto (copypasta)

**Review by jesusbv — APPROVED** [review]
LGTM 

Some typos and a question

File: integration_test-process.txt (hunk 5 above)
Comment (rjschwei) [reply integration_test-process.txt:211]: Fixed, thanks

File: integration_test-process.txt (hunk 4 above)
Comment (rjschwei) [reply integration_test-process.txt:201]: I am not sure if the docker daemon and podman can run at the same time but that has nothing to do with us. SO stopping docker avoids the question/problem ;)

**Pull request merged**